2019-07-24-HW-FileReader/src/Demo1.java

```diff
@@ -0,0 +1,84 @@
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Demo1 {
+	
+	public static void main(String[] args) {
+
+		sumOfNumbers(); // #1
+		
+		List<String> s = Arrays.asList("aaa","bbbb","a","bb");
+		System.out.println("2) Sorted strings: " + sortString(s)); // #2
+		
+		sortByAge(); // #3
+		
+		Customer c1 = new Customer("Chip", "Burunduk", 2);
+		Customer c2 = new Customer("Dale", "Burunduk", 2);
+		Customer c3 = new Customer("Joker", "BadGuy", 35);
+		Customer c4 = new Customer("Batman", "Hero", 29);
+		List<Customer> list = Arrays.asList(c1, c2, c3, c4);
+		createFileWithSortedCustomers(list); // #4
+		
+	}
+	
+	public static void sumOfNumbers() { // #1
+		try (BufferedReader br1 = new BufferedReader(new FileReader("sumOfInt.txt"));) {
+			System.out.println("1) Sum of numbers: " + sumOfInt(br1));
+		} catch (Exception e) {
+			e.printStackTrace();
+		}
+	}
+	
+	public static int sumOfInt(BufferedReader br) { // #1
```

Comment:
It is worth making the signature of this method Reader-independent. I mean that it is better if it is `public static int sumOfInt(Stream<String> stream) {...`